### PR TITLE
Feature/basic info

### PR DESCRIPTION
**Pull request by laveshaparker**
## Sign up/Issue selection/Action selection work (ROUGH)
Part of "It should allow a new user to join" [card](https://trello.com/c/pPh08sJ6)

### What this does
- Sets up basic stores for actions, issues, and UI interactions (for the sign up form)
- Handles user flow between these screens
- Components to display actions/issues/signup page

### What this doesn't do
- Any validation
- Any handling of actions aside from navigating between pages

### Todo
- [ ] All styling
- [ ] Validation around Sign up form inputs
- [ ] Proper password handling in sign up form
- [ ] Documentation around UI slice of state (a nice-to-have)
- [ ] Keep track of issue and action selections from list

### Quick demo
basic_info_demo

### CODE REVIEW

File: src/models/IssueModel.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  constructor(store, {issue_id, name}) {
+    this.store = store;
+    this.issue_id = issue_id;
+    this.name = name;
+
+    this.getId = () => {
+      return this.issue_id;
+    }
+
+    this.getName = () => {
```
Comment (laveshaparker) [inline src/models/IssueModel.js:17]: I haven't decided if I should mark this as `@computed` yet. I doubt this info would change due to user input, so probably not.

**Comment by mattmallon** [comment]
Thanks Vesha! Made me think of a couple more checklist items to add to the trello card around validation and accessibility, if you are down to keep on working on this feature branch next time you get a chance! Once the global CSS is ready (hopefully within the next week or so), it should make some of the todos easier, like clicking/tapping on issues and actions in the list to style it as selected. Thanks for the great head start here!

File: src/models/IssueModel.js (hunk 1 above)
Comment (mattmallon) [reply src/models/IssueModel.js:17]: Nahhh, issues will probably be globally defined from the database. I don't think it would change on the front-end.

**Comment by laveshaparker** [comment]
Cool! I will send out another review with styles and actions incorporated soon.

**Pull request merged**